cnf-app-mac-operator/config/rbac/role.yaml

```diff
@@ -1,11 +1,9 @@
 
 ---
 apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
+kind: ClusterRole
```

Comment:
Thanks for working on this Ramon,  I just have a question here: Is it really required to use a `ClusterRole` before it was a `Role`, afaik we are addressing the SA name, then it should work with a Role right?

Reply:
The thing is, we were not using this Role before, another one was created in this Makefile line with controller-gen binary: https://github.com/openshift-kni/example-cnf/blob/main/cnf-app-mac-operator/Makefile#L86, and that was overriding this Role.
May we use a Role instead of a ClusterRole? Probably yes, it should work, but for this first change, I'd like to follow the same approach than in the other operators, which are also cluster-wide operators, and are using ClusterRole and ClusterRoleBindings. Then, if we really need to remove Cluster* resources, we can do it in a separate change, but doing the same for all the cases.

Reply:
Thanks for the clarification, if there are other example-cnf resources with Cluster wide roles it's fine, maybe at the end we can do a security checking and see if we can change all the Cluster* stuff with Roles per namespace.